Kconfig

```diff
@@ -12,9 +12,22 @@ config BASE_OS
 source "tools/build/Kconfig.$BASE_OS"
 
 menu "Basic"
+choice LIB_OUTPUT
+	prompt "Soletta library output"
+	default SHARED_LIBRARY
+
+config SHARED_LIBRARY
+	bool "Shared"
+	depends on FEATURE_DYNAMIC_LINKER
+
+config STATIC_LIBRARY
+	bool "Static"
+endchoice
+
 # backward compatibility with autotools vars
 config ENABLE_DYNAMIC_MODULES
 	bool
+	depends on SHARED_LIBRARY
```

Comment:
this is not true. You could have a static library (libsoletta.a) that still calls `dlopen()` and loads modules as `.so`.

What you really depend on is the `FEATURE_DYNAMIC_LINKER`